.github/workflows/coverage.yml

```diff
@@ -0,0 +1,27 @@
+name: Coverage
+
+on: [pull_request]
+
+jobs:
+  pytest:
+    name: Test - pytest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set timezone
+        uses: szenius/set-timezone@v1.2
+        with:
+          timezoneLinux: "Europe/Berlin"
```

Comment:
Is there a reason this isn't Europe/Warsaw?

Reply:
Sorry, I forgot it's the first half of the 21st century. It is fixed now